ui/angular.json

```diff
@@ -42,23 +42,23 @@
             "local": {
               "fileReplacements": [
                 {
-                  "replace": "src/environments/environment.ts",
+                  "replace": "src/environments/environment.localtest.ts",
```

Comment:
This should probably not be changed, you probably need to keep the environment.ts file (perhaps has an empty stub) and in the `dev` block above, replace it with `localtest`